File: kyw0716/level0/짝수홀수 개수.js
function solution(num_list) {
  const answer = [0, 0];

  for (let i = 0; i < num_list.length; i++) {
    if (num_list[i] % 2 === 0) {
      answer[0]++;
      continue;
    }

    answer[1]++;
  }

  return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120824

File: sk000801/입문/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] gaesu = new int[2];
        for(int i = 0; i < num_list.length; i++) {
            if(num_list[i]%2==0) gaesu[0]++;
            else gaesu[1]++;
        }
        return gaesu;
    }
}